wled00/espnow_broadcast.cpp

```diff
@@ -0,0 +1,249 @@
+
+#ifndef WLED_DISABLE_ESPNOW_NEW
+#include <espnow_broadcast.h>
+
+#if defined ESP32
+#include <WiFi.h>
+#include <esp_wifi.h>
+#include <esp_now.h>
+
+#elif defined ESP8266
+#include <ESP8266WiFi.h>
+#define WIFI_MODE_STA WIFI_STA 
+#else
+#error "Unsupported platform"
+#endif //ESP32
+
+#define BROADCAST_ADDR_ARRAY_INITIALIZER {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF}
+#define WLED_ESPNOW_WIFI_CHANNEL 1
+
+typedef struct {
+    uint8_t mac[6];
+    uint8_t len;
+    uint8_t data[WLED_ESPNOW_MAX_MESSAGE_LENGTH];
+} QueuedNetworkMessage;
+static_assert(WLED_ESPNOW_MAX_MESSAGE_LENGTH <= ESP_NOW_MAX_DATA_LEN, "WLED_ESPNOW_MAX_MESSAGE_LENGTH must be <= 250 bytes");
+
+ESPNOWBroadcast espnowBroadcast;

```

Comment:
This is the 'true' name of this class
But I think we should just call it `EspNowClass` - in prep for future uses that might go beyond broadcasting?
(see comment later down about how espressif has `WifiClass`)